pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -949,6 +949,9 @@ func appNetworkDoActivateUnderlayNetwork(
 	log.Functionf("bridgeName %s MAC %s\n",
 		bridgeName, bridgeMac.String())
 
+	// get the application number on this network
+	appNumOnUNet := appNumOnUNetAllocate(ctx, netInstStatus, status.AppNum)
+
```

Comment:
We don't need to do this is ulConfig.AppIPAddr is set. But we do need to take into account that those static IP addresses get consumed from the range/subnet. So I think this logic all needs to happen inside getUlAddrs().

Reply:
Sure, moved the call into getUlAddrs(), for initialization.